app/views/index.html

```diff
@@ -18,10 +18,10 @@ <h1 class="govuk-heading-xl">
       </p>
       <ul class="govuk-list govuk-list--bullet">
         <li>
-          <a href="/laa_portal">Start an application (MVP)</a>
+          <a href="/laa_portal?mvp=true">Start an application (MVP)</a>
```

Comment:
Thoughts on removing (MVP) and (beyond MVP) brackets if we've got the mvp param?